src/datetime/_Private/util.php

```diff
@@ -0,0 +1,23 @@
+<?hh
+/*
+ *  Copyright (c) 2004-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace HH\Lib\Experimental\DateTime\_Private;
```

Comment:
We've been moving to `HH\Lib\Experimental\_Private\_DateTime` in newer stuff (I think this was originally @lexidor 's suggestion? :) )

This lets implementations easily and clearly pull in multiple private namespaces, e.g. `use namespace HH\Lib\_Private\{_DateTime,_OS};`, and makes grepping easier (as `_DateTime` can be assumed to be the private datetime NS, but `_Private` might be something else in other files)

Reply:
I have found the original conversation about this. The suggestion was yours, but I was very enthusiastic about your proposal. It is a nice best practice for files which use `_OS` and `OS`. No need to alias a use clause.